### DISCUSSION TITLE
Is correcting typos pointless?

### DISCUSSION THREAD

**Discussion by achtchaern** [monkeytypegame/monkeytype#4186] [Questions] [answered]
Is there _any_ value in correcting typos? They seem to be counted as errors, regardless of wether I correct them or not.

**Comment by Miodec** [comment, marked as answer]
When WPM is calculated, any words with errors are not counted towards your WPM (only your RAW). Correcting your mistakes will give you more WPM (the trade off being spending time correcting the mistake)

**Reply by achtchaern** [reply]
Thank you!
